src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImpl.java

```diff
@@ -29,6 +32,9 @@
 
   private final OaiHarvester harvesterOai;
 
+  @Value("${sandbox.dataset.max-size}")
+  private int maxRecords;
```

Comment:
Could not make this final, it gives an error `might not have been initialized`. I have left it static.

Reply:
Scratch that, I left this as it is. It is not possible to make it static as well with `@Value` because it will be null. Spring doesn't support `@Value` on static fields

Reply:
I figured..